frontend/src/components/Header.tsx

```diff
@@ -168,18 +167,18 @@ const Header = ({ locale }: Props) => {
         basic={true}
         className="desktop:position-sticky top-0 desktop:z-500 bg-white border-bottom-2px border-primary-vivid"
       >
-        <div className="usa-nav-container display-flex flex-align-end">
+        <div className="usa-nav-container display-flex flex-justify">
           <div className="usa-navbar border-bottom-0">
             <Title className="margin-y-2">
               <div className="display-flex flex-align-center">
-                <Link href="/" className="display-block">
-                  {logoPath && (
-                    <img
-                      src={assetPath(logoPath)}
-                      alt={t("title")}
-                      className="display-block height-4  desktop:height-auto"
-                    />
-                  )}
+                <Link href="/" className="position-relative">
```

Comment:
Next requires adding a relative position width because it automatically sets `position:absolute` to images with a `fill` prop set.